<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fsxexport_gui</class>
 <widget class="QDialog" name="fsxexport_gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>fsxexport_gui</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QCheckBox" name="TAS_checkBox">
       <property name="text">
        <string>TAS</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="IAS_checkBox">
       <property name="text">
        <string>IAS</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="DME_GPS_SPEED_checkBox">
       <property name="text">
        <string>DME/GPS_SPEED</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="BEARING_checkBox">
       <property name="text">
        <string>BEARING</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QCheckBox" name="DME_GPS_DIST_checkBox">
       <property name="text">
        <string>DME/GPS_DIST</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="GS_checkBox">
       <property name="text">
        <string>GS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="TranssCode_checkBox">
       <property name="text">
        <string>TRANSP</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="HDG_checkBox">
       <property name="text">
        <string>HDG</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="GPS_LAT_checkBox">
       <property name="text">
        <string>GPS_LAT</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="GPS_LON_checkBox">
       <property name="text">
        <string>GPS_LON</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="GPS_ALT_checkBox">
       <property name="text">
        <string>GPS_ALT</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="RADAR_ALT_checkBox">
       <property name="text">
        <string>RADAR_ALT</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="WP_ETE_checkBox">
       <property name="text">
        <string>WP_ETE</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="WP_ID_checkBox">
       <property name="text">
        <string>WP_ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="FlightPlan_lineEdit"/>
   </item>
   <item>
    <widget class="QCheckBox" name="FlightPlan_checkBox">
     <property name="text">
      <string>Flight Plan:</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Host Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostAdressEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="loginButton">
     <property name="text">
      <string>LOGIN</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
